tests/integration/test_trace_metrics.py

```diff
@@ -0,0 +1,169 @@
+from datetime import datetime, timezone
+
+from sentry_sdk.envelope import Envelope, Item, PayloadRef
+from sentry_relay.consts import DataCategory
+
+
+TEST_CONFIG = {
+    "outcomes": {
+        "emit_outcomes": True,
+        "batch_size": 1,
+        "batch_interval": 1,
+        "aggregator": {
+            "bucket_interval": 1,
+            "flush_interval": 1,
+        },
+    },
+    "aggregator": {
+        "bucket_interval": 1,
+        "initial_delay": 0,
+    },
+}
```

Comment:
```suggestion
TEST_CONFIG = {
    "outcomes": {
        "emit_outcomes": True,
    }
}
```

Changed this recently, afaik that's all that should be necessary now.